utils/mission_map.py

```diff
@@ -11,8 +10,9 @@
 names = []
 dones = []
 uniques = [] #
+extra_links = [] # List of links given by extra info
 
-for missionfile in glob.glob( './dat/missions/**/*.lua', recursive=True ):
+for missionfile in glob.glob( '../dat/missions/**/*.lua', recursive=True ):
```

Comment:
I always run it from root. It should be possible to autodetect if it is run from root or utils.

Reply:
aah. That's why. I'll take care of that